spec/poros/quote_poro_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe QuotePoro do
+  it 'will return only the info needed' do
+    quotes = QuoteService.search_by_input("extreme love")
+
+    quote_data = quotes[:results].first
+
+    quote = QuotePoro.new(quote_data)
+
+    expect(quote).to be_a QuotePoro
+    expect(quote.author).to be_a(String)
+    expect(quote.content).to be_a(String)
+    expect(quote.categories).to be_an(Array)
```

Comment:
For our poro testing, we should be testing actual stored values which is another benefit of passing data directly to our poro and skipping the real API call. We will know exactly what the expected results are instead of using `String` and `Array` which isn't very specific.